docs/guides/http/json-validation.md

```diff
@@ -0,0 +1,93 @@
+# JSON validation
+
+Bocadillo has built-in support for JSON data validation using [TypeSystem]. It allows you to perform schema-based validation, as well as use [orm] to get RESTful validation _à la_ Django REST Framework.
```

Comment:
>  schema-based validation, as well as use [orm] to get RESTful validation _à la_ Django REST Framework

Could this be dumbed down a little? Maybe it's just me but I read it a couple times and I don't really know the difference between schema and orm validation.

Reply:
Hmm. I see the reference to orm (which is an async ORM built by Tom Christie on top of TypeSystem) seems confusing.

The idea here is that TypeSystem provides schemas, so you can use them directly to perform validation. You can also directly use orm models because they’re just schemas at the core.

I’m not sure the orm thing should be mentioned in this guide at all. Maybe its place could be in the databases guide? We mention there what orm is and how to use it in Bocadillo.

Reply:
> I’m not sure the orm thing should be mentioned in this guide at all. Maybe its place could be in the databases guide? We mention there what orm is and how to use it in Bocadillo.

Yeah I think that makes sense. I think the word schema throws me off because I immediately think of a database. Is data validation really that common outside of a database? 

Maybe add something like

> If you are looking to validate data before it's saved to a database, see [link]. If you are looking to validate arbitrary JSON objects, read on!

Reply:
Sounds good, I've used this formulation. I also moved the `orm` bits to a separate `orm` how-to guide, and removed the ageing Tortoise ORM guide. :+1: